shared/src/models/utils/tag-helpers.ts

```diff
@@ -284,13 +284,13 @@ export function createBehaviorTag(
 }
 export function createPersonnelTypeTag(
     _draftState: Mutable<ExerciseState>,
-    personnelType: PersonnelType
+    personnel: Personnel | PersonnelTemplate
 ): Tag {
     return new Tag(
         'Personaltyp',
         'chocolate',
         'white',
-        personnelTypeNames[personnelType],
-        personnelType
+        personnel.name,
+        personnel.personnelType
```

Comment:
This `specifier` is just an internal identifier for aggregation and filtering. You can already use the template ID here:

```suggestion
        personnel.type === 'personnelTemplate'
            ? personnel.id
            : personnel.baseTemplateId
```